scripts/hostcfgd

```diff
@@ -781,11 +792,29 @@ class AaaCfg(object):
                 interface_ip = ipv4_addr
         return interface_ip
 
+
+    def check_file_integrity(self, filename):
```

Comment:
>check_file_integrity [](http://example.com/codeflow?start=8&length=20)

Strictly speaking, this file is only check file non-empty. If this is expected and good enough, please change the function name to narrow scope.

Otherwise, you may find some command line to verify the config. For example, `sshd -T` could be used to verify `/etc/ssh/sshd_config`. I am not sure if you can find the verification command for any other command.

Reply:
Fixed by change function name.